core-lib/src/main/AndroidManifest.xml

```diff
@@ -16,9 +16,12 @@
             android:configChanges="orientation|keyboardHidden"
             android:screenOrientation="portrait">
             <intent-filter>
+                <!--Bundle intent-->
                 <action android:name="org.idpass.smartscanner.MRZ_SCAN" />
                 <action android:name="org.idpass.smartscanner.BARCODE_SCAN" />
                 <action android:name="org.idpass.smartscanner.IDPASS_LITE_SCAN" />
+                <action android:name="org.idpass.smartscanner.QRCODE_SCAN" />
```

Comment:
we need `org.idpass.smartscanner.QRCODE_SCAN` and `org.idpass.smartscanner.odk.QRCODE_SCAN` for consistency